apis/kubedb/v1alpha2/types.go

```diff
@@ -139,3 +139,28 @@ type CoordinatorSpec struct {
 	// +optional
 	SecurityContext *core.SecurityContext `json:"securityContext,omitempty"`
 }
+
+// HealthCheckSpec defines attributes of the health check
+type HealthCheckSpec struct {
+	// How often (in seconds) to perform the health check.
+	// Default to 10 seconds. Minimum value is 1.
+	// +optional
+	// +kubebuilder:default:=10
+	PeriodSeconds *int32 `json:"periodSeconds,omitempty"`
+	// Number of seconds after which the probe times out.
+	// Defaults to 10 second. Minimum value is 1.
+	// It should be less than the periodSeconds.
+	// +optional
+	// +kubebuilder:default:=10
+	TimeoutSeconds *int32 `json:"timeoutSeconds,omitempty"`
+	// Minimum consecutive failures for the health check to be considered failed after having succeeded.
```

Comment:
`Minimum consecutive failures without having succeeded, for the health checker to be considered failed.`

Reply:
The description is also taken from the probes. https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2190-L2226